### ISSUE TITLE
Keine URL angegeben / Error in GET srsTab: error

### ISSUE THREAD

**Issue by daniel94ger** [harald-herberth/nutab#20]
Hallo zusammen,

leider bekomme ich bei verschiedenen Browsern verschiedene Fehler:

Bei Safari kommt der Fehler: Keine URL angegeben
Bei Chrome: Error in GET srsTab: error

Es geht dabei um folgende Seite: https://tennis.sc-uckerath.de/team/herren/

Ich arbeite in Wordpress und habe die folgenden Code-Zeilen in den Seiten-Editor geschrieben:

[Code-nuTab.txt](https://github.com/harald-herberth/nutab/files/6165520/Code-nuTab.txt)

Hoffe, dass sich der Fehler iwie beheben lässt, würde echt gerne die Tabelle von nuLiga auf unserer Vereinsseite anzeigen lassen.

Lg Daniel

**Comment by daniel94ger** [comment]
Inzwischen wird auch auf Safari Error in GET srsTab: error angezeigt. Hab auch alles nochmal gegen gecheckt, aber ich finde den Fehler einfach nicht... Ich gehe davon aus, dass die Datei fetch_table.php nicht gefunden wird, aber der Server und Pfad in Tabellen.js ist definitiv richtig...

**Comment by harald-herberth** [comment]
Im Debugger des Browsers sehe ich eine Fehlermeldung 403 forbidden beim Zugriff auf fetch_table.
Hat bestimmt was mit WordPress zu tun. Und damit habe ich leider keine Erfahrung. Vielleicht kann jemand dazu eine Hilfe erstellen?
Ich würde sie gerne in die Beschreibung aufnehmen.

**Comment by daniel94ger** [comment]
die 403 forbidden kommt daher, dass ich grad eben das Wordpress plugin von https://github.com/mjasny/wp-nuliga geladen hab, was auf nutab basiert, aber als Wordpress plugin umgeschrieben wurde. Hier scheint aber auch ein Fehler im plugin zu sein.

**Comment by mjasny** [comment]
Das Wordpress Plugin bindet nur @harald-herberth 's nutab code automatisch auf jeder Seite ein, sodass man nur `<div class="srsTab" data-srsurl=...` als HTML einfügen braucht. Jquery+tabellen.js sind nicht notwendig manuell einzubinden.

Eventuell muss man die eingebundenen Skripte von nuTab im wordpress plugin aktualisieren. Bei mir funktioniert für Handball noch alles.

**Comment by maxvh3688** [comment]
@mjasny ich habe ein paar Fragen zu dem Thema.
kannst du mir bitte mal eine Mail schicken oder dir mein neues Issue anschauen.
max.vonhuelsen@web.de

**Comment by mjasny** [comment]
Ich dir direkt in #24 geantwortet.

**Comment by maxvh3688** [comment]
habe nochmal was dazu zurück geschrieben.
 

Maximilian von Hülsen
Pieskower Straße 1
15526 Bad Saarow
***@***.***

 
 

Gesendet: Mittwoch, 08. Dezember 2021 um 13:14 Uhr
Von: "mjasny" ***@***.***>
An: "harald-herberth/nutab" ***@***.***>
Cc: "maxvh3688" ***@***.***>, "Comment" ***@***.***>
Betreff: Re: [harald-herberth/nutab] Keine URL angegeben / Error in GET srsTab: error (#20)


 

Ich dir direkt in #24 geantwortet.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by mjasny** [comment]
Ja, ich bekomme schon noch alle E-Mails. An deiner Stelle würde ich noch meine Anschrift im oberen Beitrag löschen ;)

**Comment by mjasny** [comment]
@daniel94ger Ich habe das WP-Plugin mit dem neuesten nuTab aktualisiert, eventuell gibt es Fehlerbehebungen für Tennis Tabellen: https://github.com/mjasny/wp-nuliga/commit/9337b092c013732226b853ff4b70207804d1a3ba

**Comment by snoopotic** [comment]
Wenn ihr auch bei Strato seid. es scheint, als ob hvs-handball.liga.nu Anfragen von Strato-Servern blockiert oder Strato Anfragen an liga.nu.

Hilft offenbar aktuell nur, nutab auf einem nicht-strato-server zu hosten. ich hätte da was ;)
